cloud-functions/functions/src/index.ts

```diff
@@ -440,3 +440,50 @@ export const initiatePasswordRecovery = functions.https.onCall((body) => {
       });
   });
 });
+
+export const getVerificationCode = functions.https.onCall(async () => {
+  const axios = require('axios');
+  const axiosCookieJarSupport = require('axios-cookiejar-support').default;
+  const tough = require('tough-cookie');
+  const qs = require('querystring');
+  axiosCookieJarSupport(axios);
+
+  let config = functions.config().verif_server;
+  let url = config.url;
+
+  const cookieJar = new tough.CookieJar();
+  let instance = await axios.create({
+    jar: cookieJar,
+    withCredentials: true,
+  });
+
+  try {
+    let response = await instance.post(
+      'https://www.googleapis.com/identitytoolkit/v3/relyingparty/verifyPassword?key=' + config.key,
+      { email: config.email, password: config.password, returnSecureToken: true }
+    );
+
+    let form = { idToken: response.data.idToken };
+
+    // Get CSRF token
+    response = await instance.get(url);
+
+    response = await instance.post(url + 'session', qs.stringify(form), {
```

Comment:
Can we optionally append a '/' character here before 'session' if it dne, since we both ran into that issue?